<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Delete</class>
 <widget class="QWidget" name="Delete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: qlineargradient(spread:pad,  x1:0, x2:0, y1:0, y2:1,  
                                      stop: 0 rgb(209, 255, 202),
                                      stop: 0.495 rgb(170, 255, 127),
                                      stop: 0.505 rgb(170, 255, 127), 
                                      stop: 1 rgb(209, 255, 202));
	border: none;
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>340</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>false</bold>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: qlineargradient(spread:pad,  x1:0, x2:0, y1:0, y2:1,  
                                      stop: 0 rgb(209, 255, 202),
                                      stop: 0.495 rgb(170, 255, 127),
                                      stop: 0.505 rgb(170, 255, 127), 
                                      stop: 1 rgb(209, 255, 202));
	border: none;
	border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
font: 22pt &quot;华文中宋&quot;;</string>
   </property>
   <property name="text">
    <string>请你选择需要删除的事件：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
